plaidml/BUILD

```diff
@@ -265,5 +265,5 @@ plaidml_py_wheel(
     console_scripts = [
        "'plaidml-setup = plaidml.plaidml_setup:main'",
     ],
-    python = "py2",
+    python = "py3",
```

Comment:
I think that adding another section for python3 will be better:
```
load(
...   
    "plaidml_py_wheel",
    "plaidml_py3_wheel",
...
)
```

```
plaidml_py3_wheel(
    name = "wheel",
    srcs = [
        ":py",
        ":configs",
    ] + select({
        "//bzl:x64_windows": ["//plaidml:plaidml.dll"],
        "//conditions:default": ["//plaidml:libplaidml.so"],
    }),
    config = ":setup.cfg",
    package = "plaidml",
    platform = select({
        "//bzl:arm7-ubuntu-14.04": "linux_armv7l",
        "//bzl:darwin": "macosx_10_10_x86_64",
        "//bzl:x64_windows": "win_amd64",
        "//conditions:default": "manylinux1_x86_64",
    }),
    console_scripts = [
       "'plaidml-setup = plaidml.plaidml_setup:main'",
    ],
    python = "py3",
)
```

Reply:
¡Gracias @rlaverde! @choongng @brianretford do you approve this approach? I can make these changes myself.

Reply:
This is totally definitely the right approach - we need to be able to build both varieties

Reply:
Awesome, go ahead :) Thanks to you for sharing it!